pages/index.js

```diff
@@ -92,79 +93,79 @@ const Home = () => {
 
   return (
     <Layout>
-      <OONIRunHero href={'https://ooni.torproject.org'} />
+      <OONIRunHero href={'https://ooni.torproject.org'}  />
 
-      <Container pt={4} maxWidth={800}>
+      <Container pt={4} sx={{maxWidth: '800px !important'}}>
```

Comment:
What is the source of the change of the prop name from `maxWidth` to `sx`? Also why do we have to mark it as `!important`? Ideally we ought to investigate how to get the maxWidth to be respected without having to give it such high priority.